cmd/kubeconfig.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+
+	"github.com/okteto/okteto/cmd/utils"
+	"github.com/okteto/okteto/pkg/analytics"
+	"github.com/okteto/okteto/pkg/cmd/login"
+	"github.com/okteto/okteto/pkg/config"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+)
+
+// Kubeconfig fetch credentials for a cluster namespace
+func Kubeconfig(ctx context.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "kubeconfig",
```

Comment:
what's the objective of this command standalone? won't you want this to happen every time you login or set okteto cloud/enterprise as your context?

Reply:
We are storing the context the user is using in a different file than your local kubeconfig. This command retrieves that kubernetes context and set it in your local kubeconfig